src/bin/pager/src/main.rs

```diff
@@ -73,12 +73,12 @@ fn main() {
         }
     })
     .detach();
-    let nvme_ctrl = twizzler_async::block_on(nvme::init_nvme());
-    let len = twizzler_async::block_on(nvme_ctrl.flash_len());
+    // let nvme_ctrl = twizzler_async::block_on(nvme::init_nvme());
```

Comment:
If we don't need this code for now then just remove it and commit.  The source code control system will remember if you need to resurrect it later.